client/app/stores/message_store.coffee

```diff
@@ -606,13 +606,18 @@ class MessageStore extends Store
         if param.type is 'message' and not _conversationMemoize.size
             messages = _currentMessages
 
-        getMessage = ->
-            index0 = messages.toArray().findIndex (message) ->
-                if param.conversationID isnt 'undefined'
-                    return param.conversationID is message.get 'conversationID'
-                return messageID is message.get 'id'
-            index = param.transform index0
-            messages.get index
+        getMessage = =>
+            _newMessage = (index) ->
```

Comment:
@aenario : Several messages have the same conversationID, so I check if next message isnt port of the same conversation; If you see a better way to do it, I am all ears